vllm_spyre/v1/worker/spyre_worker.py

```diff
@@ -657,6 +666,11 @@ def _dynamic_warmup(
         prompt_len: int,
         valid_token_ids_tensor: torch.Tensor,
     ) -> None:
+        # this setting is required to mark a dimension of size 1 as dynamic
+        # for pytorch >= 2.7.1 (needed to support batch size 1 for decodes)
+        # NB: this setting is disabled at the end of this function
```

Comment:
It'd probably be best to toss this in a context manager to ensure it gets unset, since we don't currently have a test matrix set up to test different versions of torch for compatibility.

```
with torch_backed_size_oblivious():
    ...
```

Reply:
We also likely have a new chunked prefill warmup coming very soon so this would be able to be reused there